.github/workflows/build-user-config.yml

```diff
@@ -59,12 +59,16 @@ jobs:
         run: |
           echo "build_dir=$(mktemp -d)" >> $GITHUB_ENV
 
+      - name: Install yaml2json
+        run: apt-get update && apt-get install -y jq python3-pip && python3 -m pip install remarshal
```

Comment:
This doesn't match the step with the same name on line 38. Should those match?

Also, would https://github.com/actions/setup-python be better for making sure pip is installed?